times.go

```diff
@@ -57,19 +75,47 @@ var magnitudes = []struct {
 //
 // RelTime(timeInPast, timeInFuture, "earlier", "later") -> "3 weeks earlier"
 func RelTime(a, b time.Time, albl, blbl string) string {
+	return RelTimeMagnitudes(a, b, albl, blbl, defaultMagnitudes)
+}
+
+// RelTimeMagnitudes accepts a magnitudes parameter to allow custom defined units and output format.
+//
+// example:
+// magitudes:
+// {
+//		NewMagnitude(time.Second, "now", time.Second),
+//		NewMagnitude(60*time.Second, "%d seconds %s", time.Second),
+//		NewMagnitude(120*time.Second,"a minute %s", time.Second),
+//		NewMagnitude(360*time.Second, "%d minutes %s", 60*time.Second),
+// }
+// albl: earlier
+// blbl: later
+//
+// b - a                     output
+//  -130*time.Second         2 minutes later
+//   0                       now
+//   30*time.Second          30 seconds earlier
+//   80*time.Second          a minute  earlier
+//   340*time.Second         5 minutes earlier
+//   400*time.Second         undefined
+func RelTimeMagnitudes(a, b time.Time, albl, blbl string, magnitudes []Magnitude) string {
 	lbl := albl
-	diff := b.Unix() - a.Unix()
+	diff := b.Sub(a)
 
 	after := a.After(b)
 	if after {
 		lbl = blbl
-		diff = a.Unix() - b.Unix()
+		diff = a.Sub(b)
 	}
 
 	n := sort.Search(len(magnitudes), func(i int) bool {
-		return magnitudes[i].d > diff
+		return magnitudes[i].d >= diff
```

Comment:
I have a concern about this change. It changes the behavior slightly, and it's not an improvement.

Consider the following test case:

```Go
then := time.Unix(0, 0)
now0 := time.Unix(59, 0)
now1 := time.Unix(59, 999999999)
now2 := time.Unix(60, 0)
now3 := time.Unix(60, 1)
now3 := time.Unix(60, 999999999)

fmt.Println(humanize.RelTime(then, now0, "ago", "from now"))
fmt.Println(humanize.RelTime(then, now1, "ago", "from now"))
fmt.Println(humanize.RelTime(then, now2, "ago", "from now"))
fmt.Println(humanize.RelTime(then, now3, "ago", "from now"))
fmt.Println(humanize.RelTime(then, now4, "ago", "from now"))
```

With current master, the output is:

```
59 seconds ago
59 seconds ago
1 minute ago
1 minute ago
1 minute ago
```

Which makes sense. The result changes its value after every 1 second passes.

With your branch right now, the output is:

```
59 seconds ago
59 seconds ago
60 seconds ago
1 minute ago
1 minute ago
```

That means for 1 nanosecond, the output is "60 seconds" instead of "1 minute", which is inconsistent.

I can see simply changing this `>=` to `>` breaks the other tests. Based on the above, I think the original `>` here is correct, and tests should be updated to match.